go.mod

```
@@ -1,56 +1,58 @@
 module github.com/grafana/mqtt-datasource
 
-go 1.23.5
+go 1.24.1
```

```suggestion
go 1.24.4
```
Without the toolchain.